### ISSUE TITLE
Multiple "Message generated by the parser: No declaration for attribute connect_timeout of element connection . "

### ISSUE THREAD

**Issue by mlt** [pgmodeler/pgmodeler#823]
I deleted _build_ folder and re-installed things from 6a5267dc74d0ad254ce1611c15e6730228b16ece, but I'm getting the following error messages on startup

```
[5] src\xmlparser.cpp (198)
  void XMLParser::readBuffer()
    [ERR_LIBXMLERR] Error while interpreting XML buffer at line 0 column 0.
Message generated by the parser: No declaration for attribute connect_timeout of element connection . 

[4] src\xmlparser.cpp (122)
  void XMLParser::loadXMLBuffer(const QString&)
    [ERR_LIBXMLERR] Error while interpreting XML buffer at line 0 column 0.
Message generated by the parser: No declaration for attribute connect_timeout of element connection . 

[3] src\xmlparser.cpp (91)
  void XMLParser::loadXMLFile(const QString&)
    [ERR_LIBXMLERR] Error while interpreting XML buffer at line 0 column 0.
Message generated by the parser: No declaration for attribute connect_timeout of element connection . 

[2] src\baseconfigwidget.cpp (181)
  void BaseConfigWidget::loadConfiguration(const QString&, std::map<QString, std::map<QString, QString> >&, const std::vector<QString>&)
    [ERR_LIBXMLERR] Error while interpreting XML buffer at line 0 column 0.
Message generated by the parser: No declaration for attribute connect_timeout of element connection . 

[1] src\connectionsconfigwidget.cpp (143)
  virtual void ConnectionsConfigWidget::loadConfiguration()
    [ERR_LIBXMLERR] Error while interpreting XML buffer at line 0 column 0.
Message generated by the parser: No declaration for attribute connect_timeout of element connection . 

[0] src\configurationform.cpp (119)
  void ConfigurationForm::loadConfiguration()
    [ERR_CONFIG_NOT_LOADED] Unable to load one or more configuration files! Please check if files exists in the configuration folder and if they are not corrupted to preventing this error to occur again on the next startup!
```

I tried both _develop_ and _0.8.2_ branches (e7d2bafc85dd9d03a463e0abcea6b42da6a1654f).

**Comment by rkhaotix** [comment]
The file ```connections.conf``` changed in the new release (0.8.2-beta1) and if you're upgrading an installation on your machine it's probably that you' re using the old version of the mentioned file.

__The solution__: look for the file ```connections.conf``` in your local settings storage.

* Linux: ```/home/[user]/.config/pgmodeler```
* Windows: ```C:\User\[user]\AppData\Local\pgmodeler```
* Mac OS X: ```/User/[user]/Library/Preferences/br.com.pgmodeler```

Open the file in a text editor and replace any occurrence of ```connect_timeout``` by ```connection-timeout```. Start the application again and it should stop to raise the errors.

**Comment by mlt** [comment]
Oic... I forgot about that configuration path. Thank you!
I guess [conf/connections.conf](https://github.com/pgmodeler/pgmodeler/blob/develop/conf/connections.conf#L8) should be cleaned up as well though.

**Comment by danger89** [comment]
Thanks this small change in the connection.conf file fixed it. It could be a easy fix on the 0.8.2 branch right?
